packages/patrol_cli/lib/src/ios/ios_test_backend.dart

```diff
@@ -149,8 +181,6 @@ class IOSTestBackend {
   }
 
   Future<void> uninstall(String appId, Device device) async {
-    _logger.info('Uninstalling $appId from ${device.name}');
```

Comment:
Why? :/

Reply:
If `--verbose` flag is passed to `patrol_cli`, the executed subprocesses (in this case, app uninstallation commands) are logged automatically by [LoggingLocalProcessManager from here](https://github.com/leancodepl/patrol/blob/master/packages/patrol_cli/lib/src/base/process.dart).

So, these lines I removed were unnecessary duplicates.